src/index.css

```diff
@@ -1,27 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+html,
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+}
+
+.active {
+  opacity: 1 !important;
+}
+
+.book {
+  display: flex;
+  border: 1px solid #e8e8ee;
+  justify-content: space-between;
+  padding: 2.5rem;
+  flex-direction: column;
+  margin: 1rem 0;
+}
+
+.btn {
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  font-size: 14px;
+  font-weight: 700;
+  letter-spacing: 0.5px;
+  border: none;
+  border-radius: 3px;
+  height: 58px;
+  min-width: 100px;
+  color: white;
+  margin-left: 1%;
+  cursor: pointer;
+}
+
+.btn_book {
+  background: none;
+  border: none;
+  color: #4386bf;
+  font-family: 'Roboto Slab', serif;
+  font-size: 14px;
+  cursor: pointer;
+}
+
+p {
+  color: #4386bf;
+}
+
+li {
+  border: 1px solid #e8e8ee;
+  padding: 2.5rem;
+  margin: 2rem 0;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+label {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  color: #121212;
+  opacity: 0.5;
+  margin: 0;
+}
+
+.category_p {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  color: #121212;
+  opacity: 0.5;
+  margin: 0;
+}
+
+.title {
+  font-family: 'Roboto Slab', serif;
+  margin: 0;
+  color: #4386bf;
+}
+
+.author {
+  font-family: 'Roboto Slab', serif;
+  color: #4386bf;
+  margin: 0;
+}
```

Comment:
Kindly use DRY principles for your CSS. Here values are repeating, check this [link](https://speckyboy.com/good-bad-css-practices/) to learn how you can make your CSS DRYer.